### ISSUE TITLE
Documentation should make it clear this is for the older incompatible version of libpcre

### ISSUE THREAD

**Issue by plicease** [Perl5-Alien/Alien-libpcre#7]
Also maybe track down where this gets used, from memory it is a dynamic prereq of another alien.  Might be `Alien::libnewrelic`

**Comment by plicease** [comment]
yup:

```
❌ starscream% grep Alien::libpcre */alienfile
Alien-libnewrelic/alienfile:  requires 'Alien::libpcre';
```

**Comment by plicease** [comment]
Fixed in 0.02.
